bundles/k3d-standard/uds-bundle.yaml

```diff
@@ -39,6 +50,30 @@ packages:
     ref: 0.12.0
     # x-release-please-end
     overrides:
+      loki:
```

Comment:
Not sure where this should live, but I think it would be good to document these "required" overrides in the repo somewhere.

Reply:
Updated the README.md for that bundle - https://github.com/defenseunicorns/uds-core/pull/156/files#diff-0bb4d161ebc7c1143b9b02a825ca72a61e16aec663414b221d1e0f1c3f7a0b04